tensorflow/tools/tf_windows_build_dockerfiles/Dockerfile

```diff
@@ -0,0 +1,132 @@
+# This Dockerfile creates an image that:
+# - Has the correct MTU setting for networking from inside the container to work.
+# - Has Visual Studio 2019 Build Tools installed.
+# - Has MSVC 14.29 and clang 14.0 installed
+# - Has msys2 + git, curl, zip, unzip installed.
+# - Has Python 3.9 installed.
+# - Has Bazelisk 0.11 installed.
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+SHELL ["powershell.exe", "-ExecutionPolicy", "Bypass", "-Command", "$ErrorActionPreference='Stop'; $ProgressPreference='SilentlyContinue'; $VerbosePreference = 'Continue';"]
+
+RUN netsh interface ipv4 set subinterface \"vEthernet (Ethernet)\" mtu=1460 store=persistent
+
+RUN md C:\TEMP
+RUN md C:\TMP
```

Comment:
Are we able to get rid of one of these temp directories?

Reply:
C:\TEMP was removed and tested and it was found that C:\TMP is empty when running Docker.